css/style.css

```diff
@@ -0,0 +1,133 @@
+:root{
+    --white-text:#fff;
+    --dark-text:#303030;
+    --gray-text:#686868;
+    --borders:#929292;
+    --primary-bg:#8e7754;
+    --secondary-bg:#403d38;
+    --footer-bg:#353330;
+}
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+body{
+    font-family: Alegreya Sans,sans-serif,monospace;
+    outline: none;
+}
+img{
+    display: block;
+    width: 100%;
+}
+a{
+    text-decoration: none;
+}
+li{
+    list-style: none;
+}
+/***** general *****/
+.container {
+    width: 90%;
+    margin: 0 auto;
+    max-width: 1440px;
+}
+.flexContainer {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+.paddingY {
+    padding: 2.5rem 0;
+}
+.btn{
+    padding: 0.9375rem 1.25rem;
+    color: var(--white-text);
+    cursor: pointer;
+    font-size: 1rem;
+    border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -o-border-radius: 5px;
+}
+.title{
+    color: #303030;
+    text-align: center;
+    margin-bottom: 1.875rem;
+}
+.titleDesc{
+    color: #686868;
+    line-height: 1.86em;
+    margin-bottom: 1.25rem;
+}
+.icon{
+    border: 0.5px solid #929292;
+    padding: 0.625rem;
+    margin-right: 0.3125rem;
+    color: var(--primary-bg);
+    border-radius: 50%;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    -o-border-radius: 50%;
+}
+/***** End general *****/
+/***** navbar *****/
+.navbar{
+    background: var(--white-text);
+    padding: 0.8125rem 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
+.navbar .flexContainer{
+    justify-content: flex-end;
+}
+.navHeader{
+    display: flex;
+    justify-content: flex-start;
+}
+.navToggle{
+    color: var(--dark-text);
+    background: rgba(0, 0, 0, 0);
+    border: none;
+    font-size: 1.5625rem;
+}
+.navLink a{
+    color: var(--white-text);
+    text-transform: capitalize;
+}
+.navLinks{
+    display: none;
+}
+.navLogo{
+    display: none;
+}
+@media only screen and (min-width:1024px) {
+    .navLinks{
+        display: block;
+        display: flex;
+    }
+    .navLink{
+        padding: 1.875rem 0.9375rem;
+    }
+    .navLink:first-child{
+        border-top: 2px solid var(--primary-bg);
+    }
+    .navLogo{
+        display: block;
+        width: 9.375rem;
+    }
+    .navToggle{
+        display: none;
+    }
+    .navbar .flexContainer {
+        justify-content: space-between;
+    }
+    .navbar{
+        padding: 0;
+        background: rgba(0, 0, 0, 0);
```

Comment:
can't you just use the default white here? why rgba

Reply:
i wanted it to be transparent